test/src/tests/generated/additional-properties/pets.go

```diff
@@ -31,6 +32,11 @@ func NewPetsClientWithBaseURI(baseURI string) PetsClient {
 
 // CreateAPInProperties create a Pet which contains more properties than what is defined.
 func (client PetsClient) CreateAPInProperties(ctx context.Context, createParameters PetAPInProperties) (result PetAPInProperties, err error) {
+	ctx = tracing.StartSpan(ctx, "generated/additional-properties/PetsClient.CreateAPInProperties")
+	defer func() {
+		sc := result.StatusCode
```

Comment:
We don't need a nil check here?

Reply:
yes discovered this while testing, will add a nill check and use resp instead of result